expander.go

```diff
@@ -904,7 +995,13 @@ func transitiveResolver(basePath string, ref Ref, resolver *schemaLoader) (*sche
 		rootURL.Fragment = ""
 		root, _ := resolver.cache.Get(rootURL.String())
 		var err error
-		resolver, err = defaultSchemaLoader(root, resolver.options, resolver.cache)
+
+		// shallow copy of resolver options to set a new RelativeBase when
```

Comment:
This is the new transitive: changing root with a shallow clone of options to set a new relative base